base/shell/explorer/trayntfy.cpp

```diff
@@ -124,11 +124,6 @@ class CTrayNotifyWnd :
         return TRUE;
     }
 
-    LRESULT OnMouseActivate(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
-    {
-        return MA_NOACTIVATE;
```

Comment:
This was HACK.

Reply:
Please say in your commit log that this reverts commit 83442919e :
https://github.com/reactos/reactos/commit/83442919e4df0067a7bc72f351fefed667c1737a